internal/controller/deployment_to_pdb_controller.go

```diff
@@ -57,6 +63,20 @@ func (r *DeploymentToPDBReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 	// Increment deployment count for metrics
 	metrics.DeploymentGauge.WithLabelValues(deployment.Namespace, metrics.CanCreatePDBStr).Inc()
 
+	// Check PDB_CREATE env variable
+	pdbCreate := strings.ToLower(os.Getenv("PDB_CREATE"))
+	fmt.Println("PDB_CREATE:", pdbCreate)
+	if pdbCreate != "true" {
+		// Skip PDB creation if not explicitly enabled
+		return reconcile.Result{}, nil
```

Comment:
another way to do this is to not add this reconciler at all to the manager if this is set.